### PR TITLE
feat: Add block sync monitoring script

### LINKED ISSUE #90: [FEATURE] Monitoring block synchronization
## Is your feature request related to a problem? Please describe.

In general, block synchronization should be monitored in node operations.
We could add monitoring of block synchronization as a standard tool.

## Describe the solution you'd like

- Add a monitoring script for block synchronization.
- Start the monitoring script with run.sh. When the synchronization stops, it will output an error message to the standard output.

### PR DESCRIPTION
**Pull request by YoshihitoAso**
close #90

### CODE REVIEW

File: ibet-for-fin-network/general/monitoring/monitor_block_sync.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        if latest_block_number - start_block_number > MINIMUM_INCREMENTAL_NUMBER:  # Normal
+            logging.info(
+                f"Blocks are successfully synchronized: "
+                f"start={start_block_number}, "
+                f"latest={latest_block_number}"
+            )
+            start_block_number = latest_block_number
+        else:  # Fatal Error
+            logging.error(
+                f"FATAL: Block number has not been increased: "
```
Comment (YoshihitoAso) [inline ibet-for-fin-network/general/monitoring/monitor_block_sync.py:70]: It is assumed that the operational monitoring of Quorum nodes usually does not monitor "ERROR" messages, but monitors "FATAL" messages.
Therefore, the string "FATAL:" is added to the message.

File: ibet-for-fin-network/general/Dockerfile [hunk 2]
```diff
@@ -15,12 +15,22 @@ RUN cd quorum/ && \
     mv build/bin/bootnode /usr/local/bin
 
 # Pull Geth into a second stage deploy alpine container
-FROM alpine:3.14
+FROM python:3.9-alpine3.14
```
Comment (YoshihitoAso) [inline ibet-for-fin-network/general/Dockerfile:18]: I have chosen to use the official python alpine image.

**Comment by YoshihitoAso** [comment]
sample log
```
INFO [02-19|12:31:27.407] Allocated cache and file handles         database=/eth/geth/lightchaindata cache=16.00MiB handles=16
INFO [02-19|12:31:27.477] Persisted trie from memory database      nodes=7 size=967.00B time=48.3µs gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
INFO [02-19|12:31:27.546] Successfully wrote genesis state         database=lightchaindata           hash=307166…4ad445
WARNING [2022-02-19 12:31:28,252|MONITOR-BLOCK-SYNC] Unable to connect to node
INFO [2022-02-19 12:31:58,244|MONITOR-BLOCK-SYNC] Blocks are successfully synchronized: start=0, latest=6884
INFO [2022-02-19 12:32:28,210|MONITOR-BLOCK-SYNC] Blocks are successfully synchronized: start=6884, latest=6914
INFO [2022-02-19 12:32:58,210|MONITOR-BLOCK-SYNC] Blocks are successfully synchronized: start=6914, latest=6942
ERROR [2022-02-19 12:33:28,202|MONITOR-BLOCK-SYNC] FATAL: Block number has not been increased: start=6942, latest=6942
ERROR [2022-02-19 12:33:58,171|MONITOR-BLOCK-SYNC] FATAL: Block number has not been increased: start=6942, latest=6942
INFO [2022-02-19 12:34:28,166|MONITOR-BLOCK-SYNC] Blocks are successfully synchronized: start=6942, latest=6970
INFO [2022-02-19 12:34:58,144|MONITOR-BLOCK-SYNC] Blocks are successfully synchronized: start=6970, latest=7000
```

**Comment by urushibata-k** [comment]
Needs healthcheck the monitor process.

**Review by urushibata-k — APPROVED** [review]

**Pull request merged**